sessions/js-conditions-and-booleans/teenager-check_solution/css/styles.css

```diff
@@ -1,15 +1,11 @@
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-}
-
-body {
-  font-family: system-ui;
-}
+@import './global.css';
 
 form {
-  display: grid;
-  gap: 8px;
-  max-width: 400px;
+    display: grid;
+    gap: 8px;
+    max-width: 400px;
 }
+
+#teenager-check-heading {
```

Comment:
we teach the students not to use id selectors for styling. I think we should stick to our own rules here and use a class instead.